presidio-analyzer/analyzer/logger.py

```diff
@@ -0,0 +1,74 @@
+import logging
```

Comment:
I don't see why we need this class. First, not all modules use it. Second, Python best practices is to define logger configuration once (e.g. on \_\_main\_\_) and the use `logger = logging.getLogger(__name__)` on each module. Right now we have this class + multiple logger configurations + calls to the logging module directly. Let's try to standardize everything.

Reply:
I totally agree, but standardising **everything** should be in a different pr. It's a different concern, which must be taken care of. I will open a new task for it

Reply:
This file is new and was introduced in this PR. If we do or don't standardize everything, I don't believe we need it.

Reply:
agree. If this can be resolved at the __main__ level it would be better.

Reply:
The practice on the "wrapper" class is to give you the ability to change the logging mechanism instantly across the the app, without changing the app.
 
It looks like there are many practices about how to use the "logging" module, and using the name of the module `logger = logging.getLogger(__name__)` is one of them.

The wrapper class initialises the correct way. It can be extended to provide the `getLogger` functionality easily, and let other class do it - although in Presidio, no code is doing it anyway. And I don't see any reason to do it in this "explainability" PR. It deserves a separate PR.

The current suggested change was required because it was not possible to use different loggers, due to wrong instantiation of the logger in the analyzer_engine code. 
I **Had** to introduce a the minimal code change required to have 2 different loggers. Wrapping the logging module was the only way to do it, without radical code changes.

The change is at point that:
1. The logging and tracing works, and the PR has minimal code changes for a single iteration.
2. There are many places in the python files which still use logging module, and it needs to be changed, and aligned to the wrapper logger. This tho, is not critical to the PR, and because of the nature of it - that it includes so many files, I suggest to make the changes in a dedicated PR, and not in the Tracing PR, because it crosses the gentle line of separation of concerns.